tests/formats/test_pkg.py

```diff
@@ -39,16 +33,16 @@
     "packs/players/common_fusion.pkg",
 ]
 
-@pytest.mark.parametrize("pkg_path", all_dread_pkg)
-def test_compare_dread(dread_path, pkg_path):
-    parse_and_build_compare(
-        Pkg.construct_class(Game.DREAD), Game.DREAD, dread_path.joinpath(pkg_path)
-    )
+@pytest.mark.parametrize("pkg_path", dread_data.all_files_ending_with(".pkg"))
+def test_compare_dread(dread_file_tree, pkg_path):
+    parse_build_compare_editor(Pkg, dread_file_tree, pkg_path)
```

Comment:
I'd rather the Pkg tests not use the FileTreeEditor, because that one uses Pkg a lot internally